backend/service/k8s/event.go

```diff
@@ -91,15 +88,21 @@ func (s *svc) ListNamespaceEvents(ctx context.Context, clientset, cluster, names
 	}
 
 	// Note if field selector is blank it will return everything
-	eventList, err := cs.CoreV1().Events(cs.Namespace()).List(ctx, metav1.ListOptions{FieldSelector: convertTypesToFieldSelector(types)})
-	if err != nil {
-		return nil, err
+	totalEventList := []corev1.Event{}
+	fs := convertTypesToFieldSelectors(types)
+	for _, f := range fs {
+		eventList, err := cs.CoreV1().Events(cs.Namespace()).List(ctx, metav1.ListOptions{FieldSelector: f})
```

Comment:
since we're now collecting the event results of separate API calls, do we need to sort the events list by timestamp?